### DISCUSSION TITLE
Rideable Characters and their familiars/followers

### DISCUSSION THREAD

**Discussion by WesWWilson** [Saibot393/Rideable#62] [Ideas]
Please let me know if this is something I've been ignorant enough to miss.

My primary use for Rideable is to allow PCs to have their Familiars/Followers ride on them as they move around the board.  Could we have a mode where riders are aligned as a tiny image on a corner of the main token?  Additionally, when you select the ridden token, can we have a mode to select the mounted token instead of the rider?

Example:  My lizardman player has gotten a kobold as an "Animal Companion".  That companion would normally ride piggyback on the Lizardman as a backback, only coming out as a combat token when the player wants it.  But when I select the lizardman with the kobold rider, the selected token is the rider instead of the mount.  I would prefer when I target the joint token that the lizardman is selected as the target rather than the rider.

Thank you for the mod and thank you for your hard work.

**Comment by Saibot393** [comment]
There already is a familiar riding mode. Familiar riders are place on the corners of their mount/master, the default key is "J". Selecting the mount instead of the riders when clicking the rider is a godd idea, i will look into it, should be relatively easy to do.

**Comment by Saibot393** [comment]
I have added "proxy select" client setting with v2.19.0
